<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Took_a_book</class>
 <widget class="QWidget" name="Took_a_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сдать книгу</string>
  </property>
  <widget class="QLineEdit" name="fio_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите ваши данные в формате &quot;Фамилия Имя Отчество&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ready_3">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>117</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>791</width>
     <height>211</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="library_book">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>611</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Какую книгу вы хотите сдать?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ready_4">
   <property name="geometry">
    <rect>
     <x>652</x>
     <y>540</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сдать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_fourthform">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться на главную</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
